blog/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from django.contrib import admin
+from blog.models import Post, User, Commentary
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    list_display = ("is_staff", "is_active")
```

Comment:
The 'list_display' attribute for 'UserAdmin' should include fields that uniquely identify the user, such as 'username' or 'email', to provide more meaningful information in the admin interface.